kasa/interfaces/light.py

```diff
@@ -1,4 +1,64 @@
-"""Module for Device base class."""
+"""Interact with a TPLink Light.
+
+>>> from kasa import Discover
+>>>
+>>> dev = await Discover.discover_single(
+>>>     "127.0.0.3",
+>>>     username="user@example.com",
+>>>     password="great_password"
+>>> )
+>>> await dev.update()
+>>> print(dev.alias)
+Living Room Bulb
+
+Lights, like any other supported devices, can be turned on and off:
+
+>>> print(dev.is_on)
+>>> await dev.turn_on()
+>>> await dev.update()
+>>> print(dev.is_on)
+True
+
+Get the light module to interact:
+
+>>> light = dev.modules["Light"]
```

Comment:
Should we instruct to use `Module.Light` instead of using a string here?

Reply:
Done